templates/project/sdkconfig.defaults

```diff
@@ -11,8 +11,8 @@ CONFIG_ESP_SYSTEM_EVENT_TASK_STACK_SIZE=4096
 CONFIG_PTHREAD_TASK_STACK_SIZE_DEFAULT=8192
 
 CONFIG_ESP32_DEFAULT_CPU_FREQ_240=y
-CONFIG_ESPTOOLPY_FLASHFREQ_80M=y
-CONFIG_ESPTOOLPY_FLASHMODE_DIO=y
+CONFIG_ESPTOOLPY_FLASHFREQ_80M=n
+CONFIG_ESPTOOLPY_FLASHMODE_DIO=n
```

Comment:
I can't find a lot of meaningful documentation on what these mean. Can you clarify why the template should have this as `n`? The server still has these as `y` so they will diverge.